lib/stdlib/src/erl_features.erl

```diff
@@ -187,20 +200,15 @@ keyword_fun(Opts, KeywordFun) ->
                (_) -> false
             end,
     FeatureOps = lists:filter(IsFtr, Opts),
-    {AddFeatures, DelFeatures} = collect_features(FeatureOps),
+    {AddFeatures, DelFeatures, RawFtrs} = collect_features(FeatureOps),
     %% FIXME check that all features are known at this stage so we
     %% don't miss out on reporting any unknown features.
```

Comment:
Is this FIXME still relevant? If so, could you fix it in this pull request?

Reply:
No, this has been fixed in this pull request, so I'll just remove the `FIXME`